ord_schema/validate_reactions.py

```diff
@@ -0,0 +1,55 @@
+"""Runs validations on Reaction messages.
+
+Reaction protos are expected to be stored as one message per file.
+"""
+
+import glob
+import os
+import sys
+
+from absl import app
+from absl import flags
+from absl import logging
+
+from ord_schema import message_helpers
+from ord_schema import validations
+from ord_schema.proto import reaction_pb2
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('input_pattern', None,
+                    'Pattern (glob) matching input records.')
+flags.DEFINE_enum('input_format', 'pbtxt', ['pbtxt', 'json'],
+                  'Input record format.')
+flags.DEFINE_string('output', None, 'Output file for validation errors.')
+
+
+def main(argv):
+    del argv  # Only used by app.run().
+    filenames = glob.glob(FLAGS.input_pattern)
+    num_errors = 0
+    errors = []
+    for filename in filenames:
+        reaction = message_helpers.load_message(
```

Comment:
Should this get wrapped in a try/except so that if a ```filename``` can't be parsed at all, that is written to ```FLAGS.output``` as an error?

Reply:
Yes, good idea.